### ISSUE TITLE
success rate

### ISSUE THREAD

**Issue by captainzhu123** [orrivlin/Hindsight-Experience-Replay---Bit-Flipping#4]
The question of sub-goal selection, why is it self.N: instead of 0: self.N, the method you used is inconsistent with the paper, resulting in a problem with the success rate, please answer this question

**Comment by jim1124** [comment]
> The question of sub-goal selection, why is it self.N: instead of 0: self.N, the method you used is inconsistent with the paper, resulting in a problem with the success rate, please answer this question

I agree with you. It seems like in the file "HER.py", we should use 0: self.N instead of self.N, when replay the trajectory with the fake goal.
